equipment_handler/src/adcs_handler.c

```diff
@@ -1826,8 +1860,8 @@ void get_current(float *measurement, uint8_t *address, float coef) {
  * @param coef
  * 		formatted value = rawval * coef;
  */
-void get_temp(float *measurement, uint8_t *address, float coef) {
-    *measurement = coef * uint82int16(*address, *(address + 1));
+void get_temp(float *measurement, uint16_t raw, float coef) {
+    *measurement = coef * raw;
```

Comment:
Shouldn't raw be int16? I see the function name was uint82int16.

Reply:
On that note, if your tests have not read/written negative values for signed integers, it might be a good thing to do so to make sure we have not misplaced signed and unsigned integers somewhere. The same goes with floats.

Reply:
I think so. @nsorense4 could you confirm?

Reply:
Yes. I think that should be int16.